src/core/Akka.Cluster.Tests.MultiNode/TransitionSpec.cs

```diff
@@ -51,7 +52,9 @@ protected TransitionSpec(TransitionSpecConfig config) : base(config)
 
         private RoleName Leader(params RoleName[] roles)
         {
-            return roles.First();
+            // sorts the addresses and provides the address of the node with the lowest port number
+            // as that node will be the leader
+            return roles.Select(x => Tuple.Create(x, GetAddress(x).Port)).OrderBy(x => x.Item2).First().Item1;
```

Comment:
This was the fix - needed to be able to correctly pick the leader from the remaining nodes and not just assume that the first node in the list == the leader.